src/main/java/az/company/mssubscription/exception/ErrorHandler.java

```diff
@@ -0,0 +1,44 @@
+package az.company.mssubscription.exception;
+
+import az.company.mssubscription.model.constants.ErrorMessages;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import javax.servlet.http.HttpServletRequest;
+import java.time.LocalDateTime;
+
+import static org.springframework.http.HttpStatus.INTERNAL_SERVER_ERROR;
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+
+@RestControllerAdvice
+@Slf4j
+public class ErrorHandler {
```

Comment:
1. Change ErrorResponse structure 
2. I think you don't need HttpServletRequest here